### PR TITLE
Date & Time selection component

### PR DESCRIPTION
**Pull request by shackpank**
This sort of thing.

screen shot 2015-10-12 at 13 06 22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-date-time-group",
+  "version": "1.0.0",
+  "description": "Datepicker with optional time",
+  "main": "src/DateTimeGroup.jsx",
+  "scripts": {
+    "coverage": "istanbul cover _mocha -- --compilers js:babel/register --require react-tests-globals-setup test/test* && istanbul check-coverage --statements 95 --branches 95 --functions 95 --lines 95",
```
Comment (hxpaul) [inline package.json]: nice one including coverage from the outset :+1:

**Comment by cameronviner** [comment]
After running npm install and running npm start the example page gives me the following javascript error:

Uncaught Error: Invariant Violation: TimePicker.render(): A valid ReactComponent must be returned. You may have returned undefined, an array or some other invalid object.

**Comment by cameronviner** [comment]
This looks awesome,  only one minor thing to mention includeTime and the dateName options aren't documented in the readme but other than that :+1:

File: package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "scripts": {
+    "ci": "npm run lint && npm run coverage",
+    "coverage": "istanbul cover _mocha -- --compilers js:babel/register --require react-tests-globals-setup test/test* && istanbul check-coverage --statements 95 --branches 95 --functions 95 --lines 95",
+    "prepublish": "mkdir -p ./dist && babel ./src/DateTimeGroup.jsx > ./dist/DateTimeGroup.js",
+    "prestart": "ulimit -n 9999",
+    "start": "watchify -t babelify -t brfs doc/example.js -o doc/example-built -v",
+    "test": "mocha --require react-tests-globals-setup --compilers js:babel/register test/test*",
+    "lint": "make-up src test doc"
+  },
+  "browserify": {
```
Comment (lukehansell) [inline package.json]: do we require this?

**Comment by lukehansell** [comment]
how come there are no testing instructions on this PR?

File: doc/example.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        value={myDate}
+        onChange={fns.changeDate}
+        timeClassName="myClass"
+        timeLabel="Choose a time"
+        timeName="TheTime"
+        timeStart={330}
+        timeEnd={2130}
+        timeStep={15} />
+
+      <h1>Options for date</h1>
```
Comment (lukehansell) [inline doc/example.js]: switch with the above so that the linked ones are next to each other

**Comment by lukehansell** [comment]
Y U have no ticky things?!! :+1: with doc change

**Comment by lukehansell** [comment]
also can we document using v2 npm? or can we force this somehow?

**Pull request merged**